lightning/restore/restore.go

```diff
@@ -630,6 +616,16 @@ func (rc *RestoreController) runPeriodicActions(ctx context.Context, stop <-chan
 				zap.String("state", state),
 				remaining,
 			)
+		case <-glueProgressTicker.C:
+			stage := glue.StageWriting
+			estimated := metric.ReadCounter(metric.ChunkCounter.WithLabelValues(metric.ChunkStateEstimated))
+			finished := metric.ReadCounter(metric.ChunkCounter.WithLabelValues(metric.ChunkStateFinished))
+			if finished >= estimated {
+				finished = estimated
+				stage = glue.StagePostProcessing
+			}
+			rc.tidbGlue.Record("ProgressPermillage", uint64(estimated/finished*1000))
```

Comment:
can we record the estimated and finished separately. the percent can be calculated by the glue impl.

Reply:
changed in https://github.com/pingcap/tidb-lightning/pull/456/commits/e880ad1d1475596cd4e021b2b9cac5101722a400